recipes/sracat/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "0.2" %}
+{% set name = "sracat" %}
+{% set sha256 = "6253b52e0d54cc675647ef66227792d24dde315743c14ea688a038a99277391b" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+build:
+  number: 0
+
+source:
+  url: https://github.com/lanl/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+requirements:
+  build:
+    - make
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - ncbi-ngs-sdk
```

Comment:
[This](https://github.com/sourmash-bio/wort/pull/41/files#diff-720de93c99b71895fb83beb6b48e1264a3e7a1e72f2f94d7dd0e62603a16a34bR6-R22) is what I did for building it in a docker container, so I think it is missing `sra-tools` here